lux/core/series.py

```diff
@@ -138,14 +140,13 @@ def __repr__(self):
                     )
                     print(series_repr)
                     return ""
-
-                if len(self) <= 0:
-                    warnings.warn(
-                        "\nLux can not operate on an empty series.\nPlease check your input again.\n",
-                        stacklevel=2,
-                    )
-                    print(series_repr)
-                    return ""
+                # if len(self) <= 0:
```

Comment:
We can remove the comment altogether.

For some reason, I'm still seeing the Pandas warning, is this normal?
![image](https://user-images.githubusercontent.com/5554675/109386623-cd084c00-7936-11eb-8f03-08e671cab326.png)

Reply:
I see this as well.

Reply:
Resolved, I missed a push -- I'll update y'all when I push again!